src/content/translate.ts

```diff
@@ -16,7 +16,17 @@ export interface TranslateResult {
 }
 
 export async function translate(query: TranslateQuery): Promise<TranslateResult> {
-    const apiKey = await utils.getApiKey()
+    const apiKeys = await utils.getApiKey(); 
+    const keys = apiKeys.split(';'); // Split the API keys into an array
```

Comment:
How to change the delimiter character to a comma? I hope the user experience of these two plugins is consistent.

https://github.com/yetone/bob-plugin-openai-translator/blob/main/src/main.js#L9-L10

Reply:
Of course, The code has bean committed.